packages/2018-neighborhood-development/src/components/MagnitudeOfUrbanCampsiteSweeps/index.js

```diff
@@ -1,25 +1,108 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { css } from 'emotion';
 
-import { CivicStoryCard, Placeholder } from '@hackoregon/component-library';
+import { CivicStoryCard, BarChart, Collapsable } from '@hackoregon/component-library';
+import { monthYear } from '@hackoregon/component-library/src/utils/formatters';
+
+import { fetchUrbanCampsiteSweepsByWeek } from '../../state/magnitude-of-urban-campsite-sweeps/actions';
+
+import {
+  isMagnitudeOfUrbanCampsiteSweepsPending,
+  catchMagnitudeOfUrbanCampsiteSweepsErrors,
+  getMagnitudeOfUrbanCampsiteSweepsData,
+  formatDateData,
+} from '../../state/magnitude-of-urban-campsite-sweeps/selectors';
+
+const cardLoading = css`
+  width: 100%;
+  padding: 50px;
+  text-align: center;
+  background: #EEE;
+`;
+
+const cardError = css`
+  width: 100%;
+  padding: 50px;
+  text-align: center;
+  background: #FDD;
+  border: 1px solid #C99;
+`;
 
 export class MagnitudeOfUrbanCampsiteSweeps extends React.Component {
   componentDidMount() {
-    // initialize data here
+    this.props.init();
   }
 
   render() {
+    const {
+      isLoading,
+      error,
+      magnitudeOfUrbanCampsiteSweeps,
+      // formattedDateData,
+    } = this.props;
+
+    if (isLoading) {
+      return <div className={cardLoading}>Loading...</div>;
+    } else if (!magnitudeOfUrbanCampsiteSweeps) {
+      return <div className={cardError}>{error ? `API ${error}` : 'Could not render Magnitude of Urban Campsite Sweeps.'}</div>;
+    }
```

Comment:
Does it make sense to use the `error` and `loading` states of CivicStoryCard here?